ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionValidator.java

```diff
@@ -299,6 +331,109 @@ public ValidationResult<TransactionInvalidReason> validateTransactionSignature(
     return ValidationResult.valid();
   }
 
+  public ValidationResult<TransactionInvalidReason> validateTransactionsBlobs(
+      final Transaction transaction) {
+
+    if (transaction.getBlobsWithCommitments().isEmpty()) {
+      return ValidationResult.invalid(
+          TransactionInvalidReason.INVALID_BLOBS,
+          "transaction blobs are empty, cannot verify without blobs");
+    }
+
+    BlobsWithCommitments blobsWithCommitments = transaction.getBlobsWithCommitments().get();
+
+    final long blobsLimit =
+        gasLimitCalculator.currentDataGasLimit()
+            / gasCalculator.dataGasUsed(blobsWithCommitments.getBlobs().size());
```

Comment:
DATA_GAS_LIMIT / DATA_GAS_PER_BLOB * blobCount;  
Is this correct? Why blobsLimit depends on the blob size? 
We would have 786432/ 2**17 * (6 blobs) =  786432 / 786432 = 1

Did I miss anything?

Reply:
Thinking about it, i'm not sure this check belongs here at all. if all the transactions sum to exceed the datagas limit for the block, the block would be invalid. a transaction using too much datagas is not invalid on its own, but only invalid in the context of the block.  I think we should remove this check.

Reply:
Agreed